pages/_error.js

```diff
@@ -0,0 +1,25 @@
+import React from 'react'
+
+class Error extends React.Component {
+  static getInitialProps({ res, err }) {
+    const statusCode = res ? res.statusCode : err ? err.statusCode : null;
+    return { statusCode }
+  }
+
+  render() {
+    return (
+        <section className="container has-text-centered">
+          <p>
+            <span>
+            {this.props.statusCode
```

Comment:
podrias hacer al inicio de este render un destructuring
```suggestion
            const { statusCode } = this.props
```